frontend/src/features/properties/map/research/hooks/useGetResearch.ts

```diff
@@ -1,49 +1,26 @@
-import axios, { AxiosError } from 'axios';
+import { AxiosError } from 'axios';
 import { useApiResearchFile } from 'hooks/pims-api/useApiResearchFile';
 import { IApiError } from 'interfaces/IApiError';
-import { useMemo } from 'react';
-import { useDispatch } from 'react-redux';
-import { hideLoading, showLoading } from 'react-redux-loading-bar';
 import { toast } from 'react-toastify';
-import { logError } from 'store/slices/network/networkSlice';
+
+import { useApiRequestWrapper } from './../../../../../hooks/pims-api/useApiRequestWrapper';
```

Comment:
nit: would prefer if we always import this from "hooks/pims-api/useApiRequestWrapper" instead of relative imports